buildSrc/src/main/kotlin/AbstractComposePublishingTask.kt

```diff
@@ -35,22 +35,32 @@ abstract class AbstractComposePublishingTask : DefaultTask() {
         }
     }
 
+    // android publication is always an OEL (published by androidx team, not jb),
+    // therefore add it unconditionally to OEL set
+    private val defaultOelTargetNames = setOf("android")
+
     fun publishMultiplatform(component: ComposeComponent) {
-        // To make OEL publishing (for android artifacts) work properly with kotlin >= 1.9.0,
+        val oelTargetNames = project.rootProject.findProject(component.path)!!
+            .findProperty("oel.publication.targetNames").let {
```

Comment:
Again, it looks like the wrong place to add this logic (it shouldn't be in _compose_ plugin)

Reply:
This line is not in compose plugin

Reply:
The task can be renamed. 
It's not compose specific

Reply:
It's not about a name but about a place - "Abstract**Compose**PublishingTask.kt"

Reply:
That's what I meant we can rename.